frontend/src/tests/components/Nav/AppNavbar.test.js

```diff
@@ -84,6 +84,23 @@ describe("AppNavbar tests", () => {
         expect(adminMenu).toBeInTheDocument();        
     });
 
+    test("renders correctly for admin(using admin for now until rider is implemented) user", async () => {
```

Comment:
No... you should wait until Rider is implemented.    There is no point in implementing this page until that is done.